lib/fastlane/plugin/appicon/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Appicon
-    VERSION = "0.9.0"
+    VERSION = "0.9.1"
```

Comment:
Please don't add a version bump as part of a feature PR, this should be separate 👍

Reply:
ok